test/e2e/actions/deploy/deploy_operator.go

```diff
@@ -94,7 +111,13 @@ func ClusterWideOperator(data *model.TestDataProvider) {
 	By("Deploy clusterwide Operator \n", func() {
 		kubecli.Apply("-k", data.Resources.GetOperatorFolder())
 		Eventually(
-			kubecli.GetPodStatus(config.DefaultOperatorNS),
+			func() string {
```

Comment:
This block and the next seem like a 1-to-1 copy of one another, might want to create new `func getPodStatus` to avoid repetition 💡

Reply:
Also QQ, I assume this is the only place we need kubecli after this change, right?

Reply:
Yeah. What do you mean 1-to-1 copy? Could you please tell which piece of code is repeating which one (like line 30-35 and 40-45)?

Reply:
It is obvious in kind of obvious in the `Files changes` tab but here you go :)
Lines 114-120 and 133-139. The look identical, might be missing something 🤷🏻‍♂️

Reply:
Done